polaris-react/src/styles/shared/_controls.scss

```diff
@@ -14,7 +14,7 @@
     border-radius: var(--p-border-radius-1);
 
     &::before {
-      content: '';
+      content: none;
```

Comment:
My understanding is this would entirely remove the element, and so we can delete this whole selector... Unless we're relying on some more specific selector to set `content: ''` as a trigger to activate the below rules... But that seems unlikely given this is a mixin.